test/ShapeCrawler.Tests.Unit.xUnit/ShapeTests.cs

```diff
@@ -137,4 +137,76 @@ public static IEnumerable<object[]> GeometryTypeTestCases()
         yield return new object[] { shapeCase1, Geometry.Rectangle };
         yield return new object[] { shapeCase2, Geometry.Ellipse };
     }
+    
+    public static IEnumerable<object[]> TestCasesGetShapeById()
```

Comment:
Please move tests to project `ShapeCrawler.Tests.Unit` and use NUnit. There is a plane to remove xUnit.

Reply:
Okay